.envrc

```diff
@@ -85,68 +95,79 @@ export PIP_DISABLE_PIP_VERSION_CHECK=on
 # increase node's memory limit, required for our webpacking
 export NODE_OPTIONS=--max-old-space-size=4096
 
-
 ### System ###
 
 for pkg in \
-    make            \
-    docker          \
-    chromedriver    \
-    pkg-config      \
-    openssl         ;
-    do
+    make \
+    docker \
+    chromedriver \
+    pkg-config \
+    openssl; do
     if ! require "$pkg"; then
         die "You seem to be missing the system dependency: ${pkg}
 Please install homebrew, and run brew bundle."
     fi
 done
 
-
 ### Git ###
 
 info "Configuring git..."
 
 make setup-git-config
 
-
 ### Python ###
 
+# we're enforcing that virtualenv be in .venv{,3}, since future tooling e.g.
+# venv-update will rely on this.
+venv_name=".venv"
+python_version="python2.7"
+
+if [ "$SENTRY_PYTHON3" = "1" ]; then
```

Comment:
I'd just test that this env var is set to something. = 1 is unnecessarily strict. You missed changing the other places from x1, so might as well just hit 2 birds with 1 stone there.

Reply:
hmm, I would prefer some kind of truthy check, otherwise if you set it to `0` it will still work.